connectivity/check/deployment.go

```diff
@@ -1043,8 +1027,8 @@ func (ct *ConnectivityTest) deploymentList() (srcList []string, dstList []string
 	}
 
 	if ct.params.IncludeConnDisruptTest {
-		srcList = append(srcList, testConnDisruptClientDeploymentName)
-		dstList = append(dstList, testConnDisruptServerDeploymentName)
+		srcList = append(srcList, testConnDisruptServerDeploymentName)
```

Comment:
Could you add a comment here explaining why `src` is the Server and `dst` is the client? (are there even better names for them given the test scenario?). I see the commit message, but it's not clear to me why this is needed for the multicluster case.

Reply:
Commend added. I agree that `src` and `dst` are not extremely clear as names, but I would not change them as consistent with the rest of the occurrences. For more context, the clients are initialized [here](https://github.com/cilium/cilium-cli/blob/main/connectivity/check/context.go#L641-L705) (`src` and `dst` are identical when not running in multicluster mode), the server deployment [here](https://github.com/cilium/cilium-cli/blob/main/connectivity/check/deployment.go#L619-L622) (using the `src` client), and the client one [here](https://github.com/cilium/cilium-cli/blob/main/connectivity/check/deployment.go#L672-L675) (using the `dst` client).